src/mysql_k8s_helpers.py

```diff
@@ -173,6 +173,10 @@ def configure_instance(self) -> None:
         try:
             super(MySQL, self).configure_instance(restart=False)
 
+            # ensure clean shutdown
+            # TODO: remove when pebble allow more time for shutdown
+            #       https://github.com/canonical/pebble/pull/190
+            self.container.exec(["pkill", "-15", "mysqld"])
```

Comment:
actually, this may become unnecessary when we use `mysqld_safe` instead of `mysqld` (when we address https://warthogs.atlassian.net/browse/DPE-1293)

Reply:
I do not think mysqld_safe will help... as pebble will kill mysqld_safe->mysqld after 5s anyway.

Reply:
AFAIK, pkill exits immediately after sending the signal to the mysqld and the next line in code is calling pebble to stop/kill mysqld: `self.container.restart("mysqld")`

Q: how do we increase/avoid 5s pebble timeout here?

Reply:
what i meant is that we will no longer need to run `self.container.restart("mysqld")` when we switch to using mysqld_safe. the `mysqlsh.configure_instance` function will invoke `RESTART` itself when it is done configuring the instance